wherehows-web/app/components/dataset-compliance.js

```diff
@@ -496,23 +507,36 @@ export default Component.extend({
       const complianceList = get(this, policyComplianceEntitiesKey);
       // A reference to the current field in the compliance list if it exists
       const currentFieldInComplianceList = complianceList.findBy('identifierField', identifierField);
-      const subjectId = fieldIdentifierTypes.subjectMember.value;
+      const subjectIdString = fieldIdentifierTypes.subjectMember.value;
+      // Some rendered identifierTypes may be masks of other underlying types, e.g. subjectId Member type is really
+      // a memberId with an attribute of isSubject in the affirmative
+      const unwrappedIdentifierType = subjectIdString === identifierType
+        ? fieldIdentifierTypes.member.value
+        : identifierType;
       const updatedEntity = Object.assign({}, currentFieldInComplianceList, {
         identifierField,
-        identifierType: subjectId === identifierType ? fieldIdentifierTypes.member.value : identifierType,
-        isSubject: subjectId === identifierType ? true : null,
+        identifierType: unwrappedIdentifierType,
+        isSubject: subjectIdString === identifierType ? true : null,
```

Comment:
Ah, not quite, I need it to be `null`, a Boolean cast will make it `false`.
The value can be a Boolean or Nullable, and from an api and semantic standpoint `null` has a different meaning from `false` i.e. it hasn't been set :) Thanks @yee-lkd